style.css

```diff
@@ -0,0 +1,1580 @@
+    color: #b3bac5;
+  }
+
+  .btn-submit {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding-left: 13px;
+    gap: 10px;
+    width: 170px;
+    height: 60px;
+    border: 1px solid #6070ff;
+    border-radius: 10px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 21px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    text-decoration: none;
+    color: #396df2;
+    margin-bottom: 50px;
+
+    /* for hover,enabled,disabled etc go to line 9 to 28 */
+  }
+}
+
+@media screen and (width > 768px) {
+  /* header style */
+
+  .menu-link {
+    display: none;
+  }
+
+  #header {
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    align-items: center;
+    background-color: white;
+  }
+
+  .item-nav {
+    display: block;
+    min-width: 20%;
+    justify-self: flex-end;
+  }
+
+  #navigation {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 142px;
+  }
+
+  .nav-list {
+    margin-right: 36px;
+  }
+
+  .nav-link {
+    font-family: 'Poppins', sans-serif;
+    font-style: medium;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    text-decoration: none;
+  }
+
+  /* presentation section  */
+
+  .presentation {
+    background-image: url('./images/deskHeader-bg.svg');
+    background-size: 97%;
+    background-position: left top;
+    background-color: white;
+    justify-content: center;
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    gap: -50px;
+    padding: 0;
+  }
+
+  .greeting {
+    width: 60%;
+    min-height: 20vh;
+    text-align: left;
+    margin-bottom: -50px;
+    margin-left: 260px;
+    font-family: 'Poppins', sans-serif;
+    font-style: regular;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: rgba(9, 30, 66, 1);
+    height: auto;
+    margin-top: -70px;
+  }
+
+  .self-description {
+    margin-bottom: 20px;
+    margin-left: 260px;
+    font-family: 'Poppins', sans-serif;
+    height: auto;
+    font-style: regular;
+    font-size: 20px;
+    line-height: 28px;
+    color: rgba(52, 69, 99, 1);
+  }
+
+  .btn-mediaConnect {
+    margin-left: 260px;
+    margin-bottom: 20px;
+    font-family: 'Poppins', sans-serif;
+    font-style: medium;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .media-icon {
+    width: 19.99px;
+    height: 19.96px;
+  }
+
+  .media-list {
+    margin-left: 260px;
+    width: 15%;
+  }
+
+  .media-list li {
+    margin-right: 20px;
+  }
+
+  /* project section style */
+
+  #section-project {
+    justify-content: center;
+    text-align: center;
+    justify-items: center;
+    row-gap: 100px;
+  }
+
+  .details-card {
+    width: 150%;
+    height: auto;
+    flex-direction: row;
+    gap: 20px;
+    background-color: rgba(255, 255, 255, 1);
```

Comment:
- Hey, please, as requested by the previous reviewer [here](https://github.com/fredo509/Portfolio_setup_mobile_first/pull/8#discussion_r1115219911), the correct background color is white. let's fix this to improve our design as much as we can.